exploration/percent-format.md

```diff
@@ -134,9 +135,10 @@ _What prior decisions and existing conditions limit the possible design?_
 
 ## Proposed Design
 
-_Describe the proposed solution. Consider syntax, formatting, errors, registry, tooling, interchange._
+Add a new, dedicated `:percent` function,
+which scales the _resolved value_ of its _operand_ by 100.
 
-TBD
+Separately, allow `:unit unit=percent` formatting, without scaling.
```

Comment:
Last material comment. I would make this suppositional and not part of the percent design.

```suggestion
This design could allow a function such as `:unit unit=percent` 
for formatting values without scaling
when the working group tackles units.
```